src/promptflow-evals/README.md

```diff
@@ -0,0 +1,20 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-tools)](https://pypi.org/project/promptflow-evals/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+## Introduction
+Evaluators are prebuilt promptflow pipelines that are designed to measure the quality of the outputs from large language models.
+The package includes
+ - Chat evaluator

```

Comment:
may have some one line description explain what these evaluator does?

Reply:
I have added more context to evaluators.